localization/english/culture/wc_cultures_l_english.yml

```
@@ -0,0 +1,402 @@
+ ancient_lore_collective_noun:1 "Lore Ancients"
+ ancient_wind:0 "Wind Ancient"
+ ancient_wind_collective_noun:1 "Wind Ancients"
+
+ #Trogg group
+ trogg_group:0 "Trogg"
+ trogg_group_collective_noun:1 "Troggs"
+ trogg:0 "Trogg"
+ trogg_collective_noun:1 "Troggs"
+ grummle:0 "Grummle"
+ grummle_collective_noun:1 "Grummles"
+
+ #Hozen group
+ hozen_group:0 "Hozen"
+ hozen_group_collective_noun:1 "Hozen"
+ forest_hozen:0 "Forest Hozen"
+ forest_hozen_collective_noun:1 "Forest Hozen"
+ jungle_hozen:0 "Jungle Hozen"
+ jungle_hozen_collective_noun:1 "Jungle Hozen"
+ mountain_hozen:0 "Mountain Hozen"
+ mountain_hozen_collective_noun:1 "Mountain Hozen"
+
+ #Goblin group
+ goblin_group:0 "Goblin"
+ goblin_group_collective_noun:1 "Goblins"
+ goblin:0 "Goblin"
+ goblin_collective_noun:1 "Goblins"
+ blackwater:0 "Blackwater"
+ blackwater_collective_noun:1 "Blackwaters"
+ gilblin:0 "Gilblin"
+ gilblin_collective_noun:1 "Gilblins"
+
+ #Kobold group
+ kobold_group:0 "Kobold"
+ kobold_group_collective_noun:1 "Kobolds"
+ kobold:0 "Kobold"
+ kobold_collective_noun:1 "Kobolds"
+ snobold:0 "Snobold"
+ snobold_collective_noun:1 "Snobolds"
+
+ #Draenei group
+ draenei_group:0 "Draenei"
+ draenei_group_collective_noun:1 "Draenei"
+ draenei:0 "Draenei"
+ draenei_collective_noun:1 "Draenei"
+
+ #Gnoll group
+ gnoll_group:0 "Gnoll"
+ gnoll_group_collective_noun:1 "Gnolls"
+ gnoll:0 "Gnoll"
+ gnoll_collective_noun:1 "Gnolls"
+ wolvar:0 "Wolvar"
+ wolvar_collective_noun:1 "Wolvar"
+
+ #Murloc group
+ murloc_group:0 "Murloc"
+ murloc_group_collective_noun:1 "Murlocs"
+ murloc:0 "Murloc"
+ murloc_collective_noun:1 "Murlocs"
+ gorloc:0 "Gorloc"
+ gorloc_collective_noun:1 "Gorlocs"
+
+ #Tuskarr group
+ tuskarr_group:0 "Tuskarr"
+ tuskarr_group_collective_noun:1 "Tuskarr"
+ tuskarr:0 "Tuskarr"
+ tuskarr_collective_noun:1 "Tuskarr"
+
+ #Shath'yar group
+ shathyar_group:0 "Shath'yar"
+ shathyar_group_collective_noun:1 "Shath'yar"
+ nraqi:0 "N'raqi"
+ nraqi_collective_noun:1 "N'raqi"
+
+ #Magnataur group
+ magnataur_group:0 "Magnataur"
+ magnataur_group_collective_noun:1 "Magnataurs"
+ magnataur:0 "Magnataur"
+ magnataur_collective_noun:1 "Magnataurs"
+
+ #Mogu group
+ mogu_group:0 "Nalak'sha"
+ mogu_group_collective_noun:1 "Nalakites"
+ mogu:0 "Mogu"
+ mogu_collective_noun:1 "Mogu"
+ saurok:0 "Saurok"
+ saurok_collective_noun:1 "Sauroks"
+
+ #Gnome group
+ gnome_group:0 "Gnome"
+ gnome_group_collective_noun:1 "Gnomes"
+ gnome:0 "Gnome"
+ gnome_collective_noun:1 "Gnomes"
+
+ #Centaur group
+ centaur_group:0 "Centaur"
+ centaur_group_collective_noun:1 "Centaur"
+ centaur:0 "Centaur"
+ centaur_collective_noun:1 "Centaur"
+
+ #Jinyu group
+ jinyu_group:0 "Jinyu"
+ jinyu_group_collective_noun:1 "Jinyu"
+ jinyu:0 "Jinyu"
+ jinyu_collective_noun:1 "Jinyu"
+ ankoan:0 "Ankoan"
+ ankoan_collective_noun:1 "Ankoan"
+
+ #Quilboar group
+ quilboar_group:0 "Quilboar"
+ quilboar_group_collective_noun:1 "Quilboar"
+ quilboar:0 "Quilboar"
+ quilboar_collective_noun:1 "Quilboar"
```

Why it's not pluralized?
```suggestion
 quilboar_group_collective_noun:1 "Quilboars"
 quilboar:0 "Quilboar"
 quilboar_collective_noun:1 "Quilboars"
```